charts/posthog/templates/events-hpa.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.web.hpa.enabled -}}
+{{- if and .Values.events.enabled .Values.events.hpa.enabled -}}
```

Comment:
Events and web are two different services running the same code. I think it make sense to split them up completely (for real)

Reply:
non-blocking

What does splitting them up mean btw

Reply:
Before this change we were using `web.hpa` values to configure the HPA of the `events` deployment. As those are two distinct services (that can scale in different ways) I think we should separate them.